### PR TITLE
[Tests] add imagemagick installation on appveyor

### LINKED ISSUE #402: Test failing on windows (see appveryor) test_thumbnail_quality
```
___________________________ test_thumbnail_quality ____________________________
tests\test_images.py:70: in test_thumbnail_quality
    image_size = os.path.getsize(image_file)
c:\python27\lib\genericpath.py:57: in getsize
    return os.stat(filename).st_size
E   WindowsError: [Error 2] The system cannot find the file specified: 'c:\\users\\appveyor\\appdata\\local\\temp\\1\\pytest-of-appveyor\\pytest-0\\test_thumbnail_quality0\\output\\test@192x256_q20.jpg'
```

### PR DESCRIPTION
**Pull request by vanadium23**
Imagemagick required for test_cli & test_thumbnail_quality
Fixes #402

### CODE REVIEW

**Comment by vanadium23** [comment]
Imagemagick has changed it's name on windows systems. so it was easy to fix.

File: lektor/imagetools.py [hunk 1]
```diff
@@ -368,7 +351,8 @@ def computed_height(source_image, format, width, actual_width, actual_height):
     # file where the target width is 100px will *not* be converted to a
     # 100x50 image, but to 100x200.
     if format == 'jpeg':
-        exif = read_exif(open(source_image, 'rb'))
+        with open(source_image, 'rb') as image_file:
```
Comment (nixjdm) [inline lektor/imagetools.py:354]: Good catch.

File: lektor/imagetools.py [hunk 2]
```diff
@@ -308,38 +308,21 @@ def find_imagemagick(im=None):
...
-                for filename in os.listdir(value):
-                    if filename.lower().startswith('imagemagick-'):
-                        exe = os.path.join(value, filename, 'convert.exe')
-                        if os.path.isfile(exe):
-                            return exe
-            except OSError:
-                continue
+    rv = locate_executable(imagemagick_exe)
+    if rv is not None:
+        return rv
```
Comment (nixjdm) [inline lektor/imagetools.py:325]: This is so much simpler.

**Comment by vanadium23** [comment]
@nixjdm sorry for grammar errors 🙈

**Comment by nixjdm** [comment]
No problem!

**Pull request merged**